config/manager/enoConfigMap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
With configMap being introduced,will the file pkg/config/config.go be required?

Reply:
Yes it is required because in the Config map we pas the configuration key value pairs and in the config.go we parse those values.